app/Models/Order.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class Order extends Model
+{
+    use HasFactory;
+    protected $table = "orders";
+
+
+    protected $fillable = [
+        "name",
+        "user_id",
+    ];
+    protected $hidden = [
+
+    ];
+    public function user()
```

Comment:
```suggestion
    public function users()
```
lấy ra nhiều thì dùng số nhiều nhé